test/test_dtd.py

```diff
@@ -58,6 +58,36 @@ def test_invalid_inputs(self):
                     self.dtd.validate(parsed), f"{name} validates but shouldn't"
                 )
 
+    def test_backwards_compatibility(self):
+        # the DTD needs to be backwards compatible as long as the major version does not change
+        versions = """
+                ras-dtd-1.0.readalong
+                ras-dtd-1.1.readalong
+            """.strip().split()
```

Comment:
That's a funny choice. Why not simply `versions = ["ras-dtd-1.0.readalong", "ras-dtd-1.1.readalong"]`?